.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "denm_en302637_3"]
 	path = asn1/raw/denm_en302637_3
 	url = https://forge.etsi.org/rep/ITS/asn1/denm_en302637_3.git
+[submodule "doc/doxygen-awesome-css"]
```

Comment:
Currently, https://ika-rwth-aachen.github.io/etsi_its_messages/ is not using the stylesheet yet. I'm guessing that you manually pushed the current state and it will update once this PR is pushed to main?